genai-perf/genai_perf/metrics/llm_metrics.py

```diff
@@ -26,22 +26,32 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from typing import List
+from typing import List, Union
 
 from genai_perf.metrics.metrics import MetricMetadata, Metrics
 
 
 class LLMMetrics(Metrics):
     """A simple dataclass that holds core LLM performance metrics."""
 
-    LLM_REQUEST_METRICS = [
+    LLM_REQUEST_TIME_METRICS = [
         MetricMetadata("time_to_first_token", "ms"),
         MetricMetadata("inter_token_latency", "ms"),
+    ]
+
+    LLM_REQUEST_THROUGHPUT_METRICS = [
```

Comment:
I'm a bit confused. Why do we need to distinguish time related metrics and throughput related metrics? Looking at the code, the only part that needs to treat them differently is when we convert time metric units:
```python
if slo_name in self._valid_time_related_names:
    self._valid_time_related_slos[slo_name] = (
        slo_value * self.MS_TO_NS_CONVERSION
    )
elif slo_name in self._valid_throughput_related_names:
    self._valid_throughput_related_slos[slo_name] = slo_value
```

Reply:
Because for time related metrics, we want our request values to be smaller than SLOs target, while for the throughput related metrics, we want our request values to be bigger than SLOs target.